packages/zoltan2/core/src/algorithms/color/Zoltan2_AlgHybridD1.hpp

```diff
@@ -0,0 +1,985 @@
+#ifndef _ZOLTAN2_ALGHYBRIDGMB_HPP_
```

Comment:
The file is Zoltan2_AlgHybridD1 so perhaps the defined symbol should be ZOLTAN2_ALGHYBRIDD1 for consistency? Btw, no need to have the B there but that's minor.